sdk/communication/communication-short-codes/src/utils/url.browser.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+const url = URL;
+const urlSearchParams = URLSearchParams;
+
+export { url as URL, urlSearchParams as URLSearchParams };
```

Comment:
Do you need to bind the variables? Could you simply `export { URL, URLSearchParams }`?

Reply:
Looked into this and only module-local declarations can be exported, so this is okay.